src/Symfony/Component/Cache/Tests/Adapter/DoctrineDbalAdapterTest.php

```diff
@@ -107,13 +104,15 @@ public function testConfigureSchemaTableExists()
     }
 
     /**
+     * @group legacy
```

Comment:
Legacy test cases are meant to be removed in the next major, I guess that's not the case for this one.
Can we remove this annotation?

Reply:
perfect, thanks